system/doc/reference_manual/expressions.xml

```diff
@@ -192,22 +201,112 @@ case {Value, Result} of
   </section>
 
   <section>
-    <title>Match</title>
-    <p>The following matches <c>Expr1</c>, a pattern, against
-      <c>Expr2</c>:</p>
+    <marker id="match_operator"></marker>
+    <title>The Match Operator</title>
+    <p>The following matches <c>Pattern</c> against
+      <c>Expr</c>:</p>
     <pre>
-Expr1 = Expr2</pre>
+Pattern = Expr</pre>
     <p>If the matching succeeds, any unbound variable in the pattern
-      becomes bound and the value of <c>Expr2</c> is returned.</p>
+    becomes bound and the value of <c>Expr</c> is returned.</p>
+    <p>If multiple match operators are applied in sequence, they will be
```

Comment:
I'd maybe add that even if there is a single match operator, its right hand side will be executed before the pattern on its left hand side is checked. So for example the following is valid:
```erlang
<<X:Y>> = begin Y = 8, <<42:8>> end
```
And it correctly binds X to 42, as Y is bound to 8 before the validity of the pattern on the left hand side is checked.

Reply:
Yes, I will add that clarification and an example.